README.md

```diff
@@ -1,2 +1,74 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
+we ask that these be put through a very lightweight design review process.
+
+"RFC" stands for "request for comments" and is used by many projects, e.g.
+rust-lang, to establish a consistent and controlled path for new features to
+enter their project.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Introduction](#cartographer-rfcs)
+  - [Table of Contents]
+  - [When should you write an RFC]
+  - [Motivation for RFCs]
+  - [Before creating an RFC]
+  - [What the process is]
+
+## When should you write an RFC
+[When should you write an RFC]: #when-should-you-write-an-rfc
+
+You should provide an RFC if you intent to make a "significant" change or
+contribution to one of Cartographer's repositories. A chance is e.g.
+"significant" if it introduces a new or modifies an existing "concept". The
+process is really very lightweight so we encourage you to err on the side of
+RFCs.
+
+## Motivation for RFCs
+[Motivation for RFCs]: #motivation-for-rfcs
+
+Before the introduction of RFCs reviewer's would sometimes spend time reading
+incoming PRs trying to find out what the problem is the author is actually
+solving. This could slow down reviews considerably which is unfortunate for both
+the reviewer and contributor.
+
+Sometimes a PR could sit in the review queue for a long time as it was modifying
+code that was undergoing a refactoring and merging it would make that
+refactoring harder. So the purpose of RFC is not so much to vet incoming feature
```

Comment:
It is also about vetting incoming features.

Reply:
Changed.